contracts/reserve/GoodReserveCDai.sol

```diff
@@ -523,14 +282,17 @@ contract GoodReserveCDai is
 		ERC20 _interestToken
 	) public returns (uint256, uint256) {
 		cERC20(cDaiAddress).mint(_daiToConvert);
-		uint256 interestInCdai =
-			_interestToken.balanceOf(address(this)) - _startingCDAIBalance;
-		uint256 gdInterestToMint =
-			getMarketMaker().mintInterest(_interestToken, interestInCdai);
-		uint256 gdExpansionToMint =
-			getMarketMaker().mintExpansion(_interestToken);
+		uint256 interestInCdai = _interestToken.balanceOf(address(this)) -
+			_startingCDAIBalance;
+		uint256 gdInterestToMint = getMarketMaker().mintInterest(
+			_interestToken,
+			interestInCdai
+		);
+		uint256 gdExpansionToMint = getMarketMaker().mintExpansion(
+			_interestToken
+		);
 		uint256 gdUBI = gdInterestToMint;
-		gdUBI = gdUBI.add(gdExpansionToMint);
+		gdUBI = gdUBI + gdExpansionToMint;
 		uint256 toMint = gdUBI;
```

Comment:
merge to single line gdubi =
toMint is not required it has no separate use than gdubi

Reply:
yeah right ,changed